tests/test_manager.py

```diff
@@ -114,7 +118,7 @@ def test_shutdown(self):
 
     def test_wait(self):
         # no-op
-        self.m.wait()
+        # self.m.wait()
```

Comment:
The test is still useful. It calls `wait` on line 152.
But previously, if you called `wait` before anything had been configured, it would do nothing. Now it actually waits. I probably stuck this line here to increase the branch coverage or something...